extension/sqlpgq/sqlpgq_functions/sqlpgq_iterativelength_bidirectional.cpp

```diff
@@ -120,10 +120,13 @@ static void IterativeLengthBidirectionalFunction(DataChunk &args, ExpressionStat
 
 		// make passes while a lane is still active
 		for (int64_t iter = 0; active; iter++) {
-            if (!IterativeLengthBidirectional(
```

Comment:
Please revert this file. The `make format-fix` and the CI don't agree on this file and it is driving me mad :)